ARMTemplates/README.md

```diff
@@ -0,0 +1,4 @@
+**Note**: 
+There are 2 types of updates when deploying your ARM templates: incremental update or a complete update. Please use **incremental update** when deploying ARM templates to create lab VM. By default, Resource Manager handles deployments as incremental updates to the resource group. The complete update will lead to deletion of the lab. 
```

Comment:
There is a property called Mode that has two possible values one can specify when deploying an ARM template: Incremental and Complete.